packages/@stylexjs/babel-plugin/__tests__/legacy/stylex-transform-legacy-shorthands-test.js

```diff
@@ -53,19 +53,19 @@ describe('Legacy-shorthand-expansion resolution', () => {
         var _inject2 = _inject;
         import stylex from 'stylex';
         _inject2(".x123j3cw{padding-top:5px}", 4000);
-        _inject2(".x1mpkggp{padding-right:5px}", 3000, ".x1mpkggp{padding-left:5px}");
+        _inject2(".x1gabggj{padding-inline-end:5px}", 3000);
```

Comment:
Maybe we should turn the feature flag on for these tests, if they're basically to test that the way we current use StyleX internally is not changed?